### ISSUE TITLE
npm problem

### ISSUE THREAD

**Issue by angeloCabezas** [trendmicro-frontend/react-sidenav#62]
I installed the library, but when I try to reference it, I get the following message : 

Severity	Code	Description	Project	File	Line	Suppression State
Error	TS7016	(TS) Could not find a declaration file for module '@trendmicro/react-sidenav'. 'C:/Users/LENOVO/source/repos/SideBarMenu/SideBarMenu/ClientApp/node_modules/@trendmicro/react-sidenav/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/trendmicro__react-sidenav` if it exists or add a new declaration (.d.ts) file containing `declare module '@trendmicro/react-sidenav';`	C:\Users\LENOVO\source\repos\SideBarMenu\SideBarMenu\ClientApp (tsconfig or jsconfig project)	C:\Users\LENOVO\source\repos\SideBarMenu\SideBarMenu\ClientApp\src\App.tsx	8

In short : I can't access to @trendmicro/react-sidenav

**Comment by mohsinht** [comment]
Same issue

**Comment by siefkenj** [comment]
This might be helpful: https://stackoverflow.com/questions/41462729/typescript-react-could-not-find-a-declaration-file-for-module-react-material
